recipes/python-termstyle/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "python-termstyle" %}
+{%set version = "0.1.10" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "f42a6bb16fbfc5e2c66d553e7ad46524ea833872f75ee5d827c15115fafc94e2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - termstyle
+
+about:
+  home: http://gfxmonk.net/dist/0install/python-termstyle.xml
+  license: BSD 3-Clause
+  # license_file: LICENSE not included in MANIFEST - see https://github.com/timbertson/termstyle/pull/9
```

Comment:
Then we can drop the link to PR ( https://github.com/timbertson/termstyle/pull/9 ) and add the license. 😄